.github/workflows/loc.yml

```
@@ -0,0 +1,24 @@
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+
+      - name: Run LOC check
+        run: python3 scripts/loc_report.py --threshold 500 --quiet
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)